### PR TITLE
Add awesome Collections pages and link

### PR DESCRIPTION
**Pull request by davanstrien**
Proposing this as another way for people to participate in Hacktoberfest by creating new collection on the Hub and adding a PR to these pages.

### CODE REVIEW

**Comment by davanstrien** [comment]
cc @osanseviero @Wauplin @lunarflu

**Comment by davanstrien** [comment]
Added a few examples so the lists were not completely empty, but hopefully, the rest will be fleshed out via community contributions!

File: README.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ This is a list of some wonderful open-source projects & applications integrated
 * [Official Course](https://huggingface.co/course) (from Hugging Face) - The official course series provided by 🤗 Hugging Face.
 * [transformers-tutorials](https://github.com/nielsrogge/transformers-tutorials) (by @nielsrogge) - Tutorials for applying multiple models on real-world datasets.
 
+## Awesome Collections 
+*As part of Hacktoberfest 2023, we are collection awesome Collections!*
```
Comment (Wauplin) [inline README.md:28]:
```suggestion
*As part of Hacktoberfest 2023, we are collecting awesome Collections!*
```

File: awesome_collections.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Awesome Hugging Face Collections
+
+<p align="center"><em>🎃 Note as part of [Hacktoberfest 2023](https://hacktoberfest.com/), we're looking for contributions to this page! If you have a collection you'd like to add, please submit a PR! See this issue for more context! #TODO add issue link 🎃</em></p>
```
Comment (Wauplin) [inline awesome_collections.md:3]: Let's not forget of it :+1:

File: awesome_collections.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+[Hugging Face Collections](https://huggingface.co/docs/hub/collections) provide a way of curating repositories from the Hub (Models, Datasets, Spaces and Papers) on a dedicated page. This page collects some examples of awesome collections!
```
Comment (Wauplin) [inline awesome_collections.md:5]:
```suggestion
[Hugging Face Collections](https://huggingface.co/docs/hub/collections) provide a way of curating repositories from the Hub (models, datasets, Spaces and papers) on a dedicated page. This page collects some examples of awesome collections! 
```

Usually only `Spaces` has a capital letter in the docs (AFAIU, to brand it more as a product?)

File: awesome_collections.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Awesome Hugging Face Collections
+
+<p align="center"><em>🎃 Note as part of [Hacktoberfest 2023](https://hacktoberfest.com/), we're looking for contributions to this page! If you have a collection you'd like to add, please submit a PR! See this issue for more context! #TODO add issue link 🎃</em></p>
+
+[Hugging Face Collections](https://huggingface.co/docs/hub/collections) provide a way of curating repositories from the Hub (Models, Datasets, Spaces and Papers) on a dedicated page. This page collects some examples of awesome collections! 
+
+## Awesome Hand Curated Hugging Face Collections
+- [Historic Multilingual Language Models](https://huggingface.co/collections/stefan-it/%F0%9F%93%9A-historic-multilingual-language-models-64f9c4f8383bbd73dddd2240) a Collection of Historic Multilingual Language Models. 
+- [Protein Design & Protein Structure Prediction](https://huggingface.co/collections/simonduerr/protein-design---protein-structure-prediction-64f9c6fda9295717466dbe8f) Interactive Demos that can be used for protein structure prediction using AlphaFold2 or RoseTTAfold2, prediction of small metal ions.
```
Comment (Wauplin) [inline awesome_collections.md:9]:
```suggestion
- [Historic Multilingual Language Models](https://huggingface.co/collections/stefan-it/%F0%9F%93%9A-historic-multilingual-language-models-64f9c4f8383bbd73dddd2240) (by [stefan-it](https://huggingface.co/stefan-it)) a Collection of Historic Multilingual Language Models. 
- [Protein Design & Protein Structure Prediction](https://huggingface.co/collections/simonduerr/protein-design---protein-structure-prediction-64f9c6fda9295717466dbe8f) (by [simonduerr](https://huggingface.co/simonduerr)) Interactive Demos that can be used for protein structure prediction using AlphaFold2 or RoseTTAfold2, prediction of small metal ions.
```

I would add a `by stefan-it` by a url to user profile, just to add more recognization to the awesome contributions (and it's already the case in other sections)

File: awesome_collections.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+## Awesome Automatically Generated Hugging Face Collections
+
+- [smol models](https://huggingface.co/collections/librarian-bots/smol-models-652032729004117947dc4f27) is a collection of models on the Hub which are smaller than 50MB created via [this notebook](https://huggingface.co/spaces/librarian-bots/tutorials/blob/main/smol_models_collection.ipynb)
```
Comment (Wauplin) [inline awesome_collections.md:13]:
```suggestion
- [smol models](https://huggingface.co/collections/librarian-bots/smol-models-652032729004117947dc4f27) (by [librarian-bots](https://huggingface.co/librarian-bots)) is a collection of models on the Hub which are smaller than 50MB created via [this notebook](https://huggingface.co/spaces/librarian-bots/tutorials/blob/main/smol_models_collection.ipynb)
```

**Review by Wauplin — COMMENTED** [review]
Thanks! I made a few nit comments but content looks good to me :)

**Review by Wauplin — APPROVED** [review]
Yes of course!

**Pull request merged**